Add info about data privacy in cloud connected AutoOps (#7210)
elastic/docs-content: deploy-manage/monitor/autoops/ec-autoops-faq.md

    :::
    ::::

> $$$data-privacy$$$**Who has access to the data gathered from my cluster?**

Reviewer: ```suggestion
$$$data-privacy$$$**Who has access to AutoOps?**
```

> :   You control access to your data. Within your organization, you assign user roles to manage access, and on Elastic's side, you can optionally grant access to support engineers for troubleshooting. 

Reviewer: ```suggestion
:   You control access to AutoOps. Within your organization, you assign user roles to manage access.
```
